Developers/Writing_Good_Commits.rst

```diff
@@ -139,6 +139,7 @@ More Examples
 -------------
 
```

Comment:
hey it appears you havent set up your git authorship correctly, you might want to check that before we merge this commit :) its something with git config, google it :)